frontend/src/components/account-settings.ts

```diff
@@ -233,11 +233,18 @@ export class AccountSettings extends LiteElement {
     }
 
     return html`<div class="grid gap-4">
-      <h1 class="text-xl font-semibold">${msg("Account settings")}</h1>
+      <h1 class="text-xl font-semibold">${msg("Account Settings")}</h1>
```

Comment:
Unrelated change, but I noticed we don't show name in account settings.

Reply:
It's there now! :D

We should give people the ability to change their name and email and stuff at a later date but good to add it like this for now! 👍